media-api/app/lib/ImageResponse.scala

```diff
@@ -48,22 +48,29 @@ object ImageResponse {
 
     val valid = ImageExtras.isValid(source \ "metadata")
 
+    val isAutoRetained = image.identifiers.contains(Config.persistenceIdentifier) || image.exports.length > 0
```

Comment:
Why not just keep this in one val? It's a little confusing.
Auto is an odd name as cropping isn't really auto - it's similar to archive?

Or potentially use `isArchived` but I would argue `image.userMetadata.exists(_.archived)` is clear enough?